src/docs/design/design-audit.mdx

```diff
@@ -0,0 +1,212 @@
+
+<ImageGrid images={[MediumSeverity]} regularWidth />
+
+A **low severity** issue is cosmetic and does not interfere with functionality or usability.
+
+<ImageGrid images={[LowSeverity]} regularWidth />
+
+In addition, auditors may suggest **features** as well. Features are good-to-have improvements; users often notice them in other Brightlayer UI apps.
+
+<ImageGrid images={[Feature]} regularWidth />
+
+As hinted above, audit notes may also be tagged with whether an issue is about usability or branding. This further helps product teams think about usability systematically.
+
+Auditors are recommended to color code their design audit notes based on severity.
+
+# How to Conduct Design Audits (For Auditors)
+
+## Prepare Deliverables
+
+A full audit done by the Brightlayer UI team typically includes two parts: detailed notes that capture everything, and a presentation slide deck including only items worth discussing.
+
+<ImageGrid
+    images={[Banner, Slides]}
+    captionsUnderImages={[
+        `The detailed audit notes.`,
+        `The presentation helps foster in-depth discussions around a given topic.`,
+    ]}
+/>
+
+Auditors are expected to build up detailed audit notes as they go through each screen. Once auditors are finished, they may consider selecting a few educational or discussion-worthy topics and adding them to a separate PowerPoint slide.
+
+## Apply Brightlayer UI’s Standard
+
+<ImageGrid
+    images={[DoDont]}
+    captionsUnderImages={[`You can find these color-coded design standards across Brightlayer UI's design guidelines.`]}
+/>
+
+There are two parts to Brightlayer UI’s design guidelines — [style guide](/style) and [design patterns](/patterns). Each of these describes a set of design standards, which are categorized as follows:
+
+<RuleTable />
+
+User interfaces must be judged under these criteria during design audits.
+
+In addition, auditors may also employ [Heuristic Evaluation](https://www.nngroup.com/articles/how-to-conduct-a-heuristic-evaluation/) to check UI designs comprehensively.
+
+## Educate the Team
+
+To help teams develop a human-centered mindset, in addition to pointing out the issues, auditors must also seek to educate their product team to think like designers.
+
+<ImageGrid
+    images={[Educational1, Educational2]}
+    captionsUnderImages={[
+        `AVOID: Avoid giving the team an instruction without explaining the rationale behind it.`,
+        `DO: Teach the team how this design decision is made.`,
+    ]}
+/>
+
+For larger products, instead of trying to cover everything in superficial detail, auditors may select 2-3 key interactions and examine them in depth. The auditors’ mission should not be to point out every single UI mistake in this case, as there are likely going to be follow-up UI audits in the future, but to teach the team how to avoid similar mistakes in the future.
```

Comment:
```suggestion
For larger products, instead of trying to cover everything in superficial detail, auditors may select 2-3 key interactions and examine them in depth. The auditors’ mission should not be to point out every single UI mistake, but to teach the team how to avoid similar mistakes in the future.
```